src/main/kotlin/com/cognifide/gradle/aem/deploy/tasks/PurgeTask.kt

```diff
@@ -0,0 +1,28 @@
+package com.cognifide.gradle.aem.deploy.tasks
+
+import com.cognifide.gradle.aem.AemPlugin
+
+open class PurgeTask : AbstractTask() {
+
+    companion object {
+        val NAME = "aemPurge"
+    }
+
+    init {
+        group = AemPlugin.TASK_GROUP
+        description = "Uninstalls and then deletes AEM package on instance(s)."
+    }
+
+    @org.gradle.api.tasks.TaskAction
+    fun install() {
+        deploy({ sync ->
+            ensureUserAwareness(NAME)
```

Comment:
okay, changed. BTW. if some CI would run it periodically, there would be a user who set up such CI job and explicitly set this force flag, and that would be the aware user here ;)